test/liquidate/liquidate.ts

```diff
@@ -149,7 +147,7 @@ describe('Liquidate Tests', () => {
       firstCluster.owner,
       firstCluster.operatorIds,
       firstCluster.cluster
-    )).to.be.revertedWithCustomError(ssvNetworkContract,'ClusterNotLiquidatable');
+    )).to.be.revertedWithCustomError(ssvNetworkContract, 'ClusterNotLiquidatable');
```

Comment:
Not sure if we have already but maybe in this test to also check `isLiquidatable` after doing `liquidate` as it should be false after liquidation

Reply:
I move the `isLiquidatable` check to this test: `Liquidate validator with removed operator in a cluster`
`isLiquidatable` reverts with `InsufficientFunds` but this is fixed in [this PR](https://github.com/bloxapp/ssv-network/pull/198)